### PR TITLE
167727624 Build server to deploy on Heroku

### PR DESCRIPTION
**Pull request by victor-shagor**
#### What does this PR do?
         implement server make the application live on Heroku
#### Description of Task to be completed?
          add index.js file
#### How should this be manually tested?
    [Visit](rogue-backend-staging.herokuapp.com)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167727624](https://www.pivotaltracker.com/story/show/167727624)
#### Questions:

### CODE REVIEW

**Review by 0xciph3r — APPROVED** [review]
The link is working. Great Job Abiola

File: src/index.js [hunk 1]
```diff
@@ -1,93 +1,18 @@
...
-    res.status(err.status || 500);
-    res.json({
-        errors: {
-            message: err.message,
-            error: {}
-        }
-    });
+const port = process.env.PORT || 3000;
+app.listen(port, () => {
+  console.log(`Listening from port ${port}`);
```
Comment (Daymorelah) [inline src/index.js]: use console.info instead.

File: src/index.js [hunk 2]
```diff
@@ -1,93 +1,18 @@
...
-app.use(cors());
-
-// Normal express config defaults
-app.use(require("morgan")("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cors())
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.get('/', (req, res) => res.status(200).send({ message: 'Welcome to Barefoot Nomad' }));
+app.use('*', (req, res) => res.send({ message: 'route not found' }));
```
Comment (Daymorelah) [inline src/index.js:11]: use app.all() instead

File: src/index.js [hunk 3]
```diff
@@ -1,93 +1,18 @@
...
-
-// Create global app object
 const app = express();
-
-app.use(cors());
-
-// Normal express config defaults
-app.use(require("morgan")("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cors())
```
Comment (Daymorelah) [inline src/index.js]: This line of code should come after setting the bodyParser middleware

**Review by ChiamakaObitube — APPROVED** [review]
Good job here @victor-shagor. I tested the heroku app link and I can see the welcome message.

File: src/index.js [hunk 4]
```diff
@@ -1,93 +1,18 @@
...
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import bodyParser from 'body-parser';
+import express from 'express';
+import cors from 'cors';
```
Comment (Daymorelah) [inline src/index.js:3]: express should be imported first.

**Review by Daymorelah — CHANGES_REQUESTED** [review]

**Review by 0xciph3r — APPROVED** [review]

**Review by Daymorelah — COMMENTED** [review]
LGTM!

**Review by Daymorelah — APPROVED** [review]

**Pull request merged**